clippy_lints failed to compile

I have this error while executing `cargo install clippy`
```
error[E0308]: mismatched types
   --> /Users/Khaled/.cargo/registry/src/github.com-1ecc6299db9ec823/clippy_lints-0.0.110/src/utils/mod.rs:579:18
    |
579 |         message: help_msg,
    |                  ^^^^^^^^ expected struct `collections::vec::Vec`, found struct `collections::string::String`
    |
    = note: expected type `collections::vec::Vec<(collections::string::String, rustc_errors::snippet::Style)>`
               found type `collections::string::String`

error: aborting due to previous error

error: failed to compile `clippy v0.0.110`, intermediate artifacts can be found at `/var/folders/1_/mgbysfpj1s36vf1hq3qm2_8c0000gn/T/cargo-install.F3BHcTC9PcJW`

Caused by:
  Could not compile `clippy_lints`.
```
rustc version: rustc 1.16.0-nightly
OS: macOS Sierra

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,11 +1,14 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## 0.0.111 — 2017-01-21
+* Rustup to *rustc 1.16.0-nightly (a52da95ce 2017-01-20)*
+
 ## 0.0.110 — 2017-01-20
 * Add badges and categories to `Cargo.toml`
 
 ## 0.0.109 — 2017-01-19
-* Update to *1.16.0-nightly (c07a6ae77 2017-01-17)*
+* Update to *rustc 1.16.0-nightly (c07a6ae77 2017-01-17)*
 
 ## 0.0.108 — 2017-01-12
 * Update to *rustc 1.16.0-nightly (2782e8f8f 2017-01-12)*

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "clippy"
-version = "0.0.110"
+version = "0.0.111"
 authors = [
 	"Manish Goregaokar <manishsmail@gmail.com>",
 	"Andre Bogus <bogusandre@gmail.com>",
@@ -30,7 +30,7 @@ test = false
 
 [dependencies]
 # begin automatic update
-clippy_lints = { version = "0.0.110", path = "clippy_lints" }
+clippy_lints = { version = "0.0.111", path = "clippy_lints" }
 # end automatic update
 
 [dev-dependencies]

--- a/clippy_lints/Cargo.toml
+++ b/clippy_lints/Cargo.toml
@@ -1,7 +1,7 @@
 [package]
 name = "clippy_lints"
 # begin automatic update
-version = "0.0.110"
+version = "0.0.111"
 # end automatic update
 authors = [
 	"Manish Goregaokar <manishsmail@gmail.com>",

